src/libcore/num/mod.rs

```diff
@@ -350,6 +371,27 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Returns the number of trailing ones in the binary representation of `self`.
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "let n = 0b10110011", stringify!($SelfT), ";
```

Comment:
Tidy is saying that you need `#![feature(leading_ones_trailing_ones)]` in the doctests.